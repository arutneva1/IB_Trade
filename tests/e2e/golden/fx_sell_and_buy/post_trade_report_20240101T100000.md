| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| USD | BUY | 65.1200 | 1.36 | 88.56 | 0.00 | -200.94 |
| BBB | SELL | -18.0000 | 49.00 | -882.00 | 0.00 | 45.18 |
| AAA | BUY | 17.0000 | 101.00 | 1717.00 | 0.00 | -245.71 |
