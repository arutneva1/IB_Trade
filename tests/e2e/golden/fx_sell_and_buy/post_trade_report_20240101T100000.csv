symbol,side,filled_shares,avg_price,notional,avg_slippage,residual_drift_bps
USD,BUY,65.12,1.36,88.56,0.0,-200.94
BBB,SELL,-18.0,49.0,-882.0,0.0,45.18
AAA,BUY,17.0,101.0,1717.0,0.0,-245.71
